.ruby-version

```diff
@@ -0,0 +1 @@
+ruby-2.3.0
```

Comment:
Use the oldest ruby version that we use. Probably the version that we use in core. That way we can ensure that we are backwards compatible.